securedrop_client/app.py

```diff
@@ -150,7 +156,14 @@ def start_app(args, qt_args) -> None:
 
 
 def run() -> None:
+    config_file = "/etc/securdrop-client/client.ini"
```

Comment:
We should not be configuring from `/etc`. Configs should be at the root of `sdc_home`.

Reply:
We should configure with `/etc/` as that will be useful to update through the Debian package. The package will be installed at the template, and we should have the default values already in the config for the same.

Reply:
I'm a little confused on the goals then. Is the `securedrop-client` so hyper targeted toward Qubes that we aren't going to work in having it operate anywhere else ever? Or should we use some patterns that allow us to be able to run it in other linux-y environments? Because what I *don't* like about SecureDrop core is that the Debian packages are useless without running `ansible` first. IMO configuration should be entirely outside the Debian side and use defaults in the Python code to handle the no-config cases.